jiant/trainer.py

```diff
@@ -851,7 +851,9 @@ def _calculate_validation_performance(
             batch_num += 1
             with torch.no_grad():
                 out = self._forward(batch, task=task)
-            loss = out["loss"]
+            if "loss" in out:
```

Comment:
when do we not have a loss ?

Reply:
I guess in some seq2seq example, but why can't/wouldn't we compute loss ?

Reply:
Nono, never when getting validation performance; this is from before when I didn't compute a loss. Changed!